### PR TITLE
Crypto-1,Crypto-13: Create Intro & Home Screens

### PR DESCRIPTION
**Pull request by vitalyaliskevych**
### What's New
- Created UI for Intro&Home Screens
### Issues
- [Crypto-1](https://trello.com/c/mGp6X5rP)
- [Crypto-13](https://trello.com/c/rkAkjIIz)
### Screenshots

| MainScreen  | IntroScreen |
| ------------- | ------------- |
| Screenshot 2022-12-23 at 18 53 02  | Screenshot 2022-12-23 at 18 55 25  |

### CODE REVIEW

File: Crypto/Flows/Screens/MainScreen/MainView.swift [hunk 1]
```diff
@@ -107,29 +78,32 @@ extension MainView {
         VStack {
             VStack {
                 Button(action: {}) {
-                        HStack {
-                            imagName
+                    HStack {
+                        imagName
```
Comment (oleksandr-kozlovskyi) [inline Crypto/Flows/Screens/MainScreen/MainView.swift:82]: rename to `imageName` please

Comment (vitalyaliskevych) [reply Crypto/Flows/Screens/MainScreen/MainView.swift:82]: Done

File: Crypto/Flows/Screens/IntroScreen/IntroView.swift [hunk 2]
```diff
@@ -39,20 +39,20 @@ extension IntroView {
     var welcome: some View {
         Text("Welcome")
             .foregroundColor(Color.white)
-            .font(.system(size: 40))
+            .font(.custom("Poppins-Bold", size: 40))
```
Comment (romansorochak) [inline Crypto/Flows/Screens/IntroScreen/IntroView.swift:42]: Better to have some enum for those fonts.

File: Crypto/Flows/Screens/MainScreen/MainView.swift [hunk 3]
```diff
@@ -41,53 +33,32 @@ struct MainView: View {
     
 }
 
-extension MainView {
+private extension MainView {
+    var cryptoText: some View {
```
Comment (romansorochak) [inline Crypto/Flows/Screens/MainScreen/MainView.swift:37]: `var titleView: ...`

File: Crypto/Flows/Screens/MainScreen/MainView.swift [hunk 4]
```diff
@@ -41,53 +33,32 @@ struct MainView: View {
...
-                //            ethBlock
-                btcBlock
-                ltcBlock
-                ethBlock
-                btcBlock
-                ltcBlock
-                ethBlock
+                
+                ForEach(viewModel.coinModels, id: \.id) { coinModel in
+                    createButton(imagName: coinModel.icon, text: coinModel.name, number: "$\(coinModel.price)")
```
Comment (romansorochak) [inline Crypto/Flows/Screens/MainScreen/MainView.swift:51]: update createButton function so it adds some padding around number.

File: Crypto/Flows/Screens/IntroScreen/IntroView.swift (hunk 2 above)
Comment (vitalyaliskevych) [reply Crypto/Flows/Screens/IntroScreen/IntroView.swift:42]: done

File: Crypto/Flows/Screens/MainScreen/MainView.swift (hunk 3 above)
Comment (vitalyaliskevych) [reply Crypto/Flows/Screens/MainScreen/MainView.swift:37]: done

File: Crypto/Flows/Screens/IntroScreen/IntroView.swift (hunk 2 above)
[the remark this answers is not in the document]
Comment (vitalyaliskevych) [reply Crypto/Flows/Screens/IntroScreen/IntroView.swift:42]: done

File: Crypto/Flows/Screens/IntroScreen/IntroView.swift [hunk 5]
```diff
@@ -39,20 +39,20 @@ extension IntroView {
...
     }
     var welcome2: some View {
         Text("To the Crypto")
             .foregroundColor(Color.white)
-            .font(.system(size: 30))
+            .font(.custom("Poppins-SemiBold", size: 30))
```
[the remark this answers is not in the document]
Comment (vitalyaliskevych) [reply Crypto/Flows/Screens/IntroScreen/IntroView.swift]: done

File: Crypto/Utilities/Coin.swift [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+//  Crypto
+//
+//  Created by Developer on 28.12.2022.
+//
+
+import SwiftUI
+
+struct Coin: Identifiable {
+    var id = UUID()
+
```
Comment (oleksandr-syurpita) [inline Crypto/Utilities/Coin.swift:12]: remove space

File: Crypto/Utilities/Models/Coin.swift [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+//  Crypto
+//
+//  Created by Developer on 14.12.2022.
+//
+
+import SwiftUI
+
+struct Coin: Identifiable {
+    var id = UUID()
+
```
Comment (oleksandr-syurpita) [inline Crypto/Utilities/Models/Coin.swift:12]: remove space

File: Crypto/Utilities/Models/CoinInfo.swift [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+//  Crypto
+//
+//  Created by Developer on 21.12.2022.
+//
+
+import SwiftUI
+
+struct CoinInfo: Identifiable {
+    var id = UUID()
+
```
Comment (oleksandr-syurpita) [inline Crypto/Utilities/Models/CoinInfo.swift:12]: remove space

Comment (vitalyaliskevych) [reply Crypto/Utilities/Models/CoinInfo.swift:12]: i remmove this file

File: Crypto/Utilities/Models/Coin.swift (hunk 7 above)
Comment (vitalyaliskevych) [reply Crypto/Utilities/Models/Coin.swift:12]: done

**Review by oleksandr-syurpita — APPROVED** [review]

**Pull request merged**